src/Scripting/CSharp/CSharpScriptingResources.resx

```diff
@@ -141,4 +141,7 @@ Options:
   --                             Indicates that the remaining arguments should not be treated as options.
 ]]></value>
   </data>
+  <data name="CannotSetCSharpLanguageVersion" xml:space="preserve">
+    <value>Cannot set C# LanguageVersion because ParseOptions were already configured for a different language.</value>
```

Comment:
>ParseOptions [](start = 49, length = 12)

The fact that we use ParseOptions is an implementation detail, so the user wouldn't know from the message what's going on here. 
I would reword this to 

```
Cannot set {0} specific option {1} because the script was already configured for a different language.
```

where {0} would be `C#` and {1} would be `nameof(LanguageVersion)`. Parameterizing the message makes it easier to translate (non-translated words are not part of the resource string) and also allows us to use the message for other language specific options if needed.

Reply:
done. given this approach, I also moved the message from `CSharpScriptingResources.resx` to `ScriptingResources.resx`